go-controller/pkg/network-controller-manager/node_network_controller_manager.go

```diff
@@ -103,6 +103,16 @@ func (ncm *nodeNetworkControllerManager) getOVNIfUpCheckMode() error {
 	return nil
 }
 
+// initDefaultNetworkController creates the controller for default network
+func (ncm *nodeNetworkControllerManager) initDefaultNodeNetworkController() error {
+	defaultNodeNetworkController, err := node.NewDefaultNodeNetworkController(ncm.newCommonNetworkControllerInfo())
```

Comment:
Maybe we could add a short comment stating why it is not done as straight assignment? for posterity :)

Reply:
> Maybe we could add a short comment stating why it is not done as straight assignment? for posterity :)

+1

Reply:
Added the comment in the two places I am aware we are doing this.